spec/models/pawn_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Pawn, type: :model do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+  let(:pawn_white) { FactoryGirl.create(:pawn, :is_on_board, color: 'white') }
+  let(:pawn_black) { FactoryGirl.create(:pawn, :is_on_board, color: 'black', row_coordinate: 1) }
+  let(:pawn_center) { FactoryGirl.create(:pawn, :is_on_board, row_coordinate: 3) }

```

Comment:
I don't think these pawns are on the same board. Need to set `game: game` to be sure.

Otherwise, factory girl is probably creating a new game for each of these.